### PR TITLE
add some docs on how to use helm-chart

### PR DESCRIPTION
**Pull request by szaimen**

### CODE REVIEW

**Review by st3iny — APPROVED** [review]
I tested helm charts instructions and they work.

**Comment by szaimen** [comment]
Thanks a lot for testing Richard! :)

File: nextcloud-aio-helm-chart/readme.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 # You can also install the AIO containers on Kubernetes using this Helm Chart
 
 This is currently beta and not ready yet.
+
+## How to use this?
+
+```
+helm repo add nextcloud-aio https://nextcloud.github.io/all-in-one/
+helm install my-release nextcloud-aio/nextcloud-aio-helm-chart
```
Comment (szaimen) [inline nextcloud-aio-helm-chart/readme.md:9]: Need to add a hint to overwrite values from https://github.com/nextcloud/all-in-one/blob/main/nextcloud-aio-helm-chart/values.yaml. Especially at least the ones marked with todo

Comment (szaimen) [reply nextcloud-aio-helm-chart/readme.md:9]: `-f my-values.yaml`

Comment (szaimen) [reply nextcloud-aio-helm-chart/readme.md]: done

**Pull request merged**